README.md

```diff
@@ -23,13 +23,37 @@ practices.
 ## Versioning
 The commit hash of the deployed code is considered its version identifier. The commit hash can be retrieved locally via `git rev-parse HEAD`.
 
-## Setting Up
-
-Contile uses Rust, and requires the latest stable iteration. See
-[rustup.rs](https://rustup.rs/) for how to install this application.
-
-Once Rust is installed you can compile using `cargo build`. This will
-create a development release.
+## Setting Up Your Local Dev Environment
+1. Install Rust. See [rustup.rs](https://rustup.rs/) for how to install on your platform.
+2. Compile Contile using `cargo build`. This will create a development release.
```

Comment:
nit: remove more unnecessary info. The title of this section is: "Setting Up Your Local Dev Environment" and "development release" is ambiguous. 
```suggestion
2. Compile Contile using `cargo build`. 
```